<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceQ</class>
 <widget class="QWidget" name="ChoiceQ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>正在答题</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QTextEdit" name="question_text">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>40</y>
     <width>750</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="A_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="B_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="C_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="D_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>500</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_btn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>⇦</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_btn">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>220</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>28</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>⇨</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="A_textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>580</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="B_textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>580</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="C_textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>400</y>
     <width>580</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="D_textEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>580</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="index_lbl">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>580</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="getAns_btn">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>530</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>查看答案</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
